.vib/external-dns/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/external-dns"
         },
-        "runtime_parameters": "ImNsb3VkZmxhcmUiOgogICJhcGlLZXkiOiAidGVzdGluZ2FwaWtleTEyMzQiCiAgImVtYWlsIjogInRlc3RAdGVzdC5vcmciCiJwcm92aWRlciI6ICJjbG91ZGZsYXJlIgoic2VydmljZSI6CiAgInBvcnRzIjoKICAgICJodHRwIjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "Y29udGFpbmVyUG9ydHM6CiAgaHR0cDogNzk4MApjbG91ZGZsYXJlOgogIGFwaUtleTogdGVzdGluZ2FwaWtleTEyMzQKICBlbWFpbDogdGVzdEB0ZXN0Lm9yZwpwcm92aWRlcjogY2xvdWRmbGFyZQppbnRlcnZhbDogIjFtIgpsb2dMZXZlbDogaW5mbwpsb2dGb3JtYXQ6IHRleHQKcG9saWN5OiB1cHNlcnQtb25seQpyZWdpc3RyeTogInR4dCIKc2VydmljZToKICBwb3J0czoKICAgIGh0dHA6IDgwCiAgdHlwZTogTG9hZEJhbGFuY2VyCnNlcnZpY2VBY2NvdW50OgogIGNyZWF0ZTogdHJ1ZQogIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWUKcmJhYzoKICBjcmVhdGU6IHRydWUKICBjbHVzdGVyUm9sZTogdHJ1ZQpwb2RTZWN1cml0eUNvbnRleHQ6CiAgZW5hYmxlZDogdHJ1ZQogIGZzR3JvdXA6IDEwMDIKICBydW5Bc1VzZXI6IDEwMDI=",
```

Comment:
I think some properties here are optional and do not really produce a different outcome on the tests:

```diff
containerPorts:
  http: 7980
cloudflare:
  apiKey: testingapikey1234
  email: test@test.org
provider: cloudflare
- interval: "1m"
- logLevel: info
- logFormat: text
- policy: upsert-only
- registry: "txt"
service:
  ports:
    http: 80
  type: LoadBalancer
serviceAccount:
  create: true
  automountServiceAccountToken: true
rbac:
  create: true
  clusterRole: true
podSecurityContext:
  enabled: true
  fsGroup: 1002
  runAsUser: 1002
```

Am I missing something?

Reply:
No, you are right. Those parameters are some leftovers that didn't get a test at the end. Removed them now, thanks!